examples/concept.js

```diff
@@ -226,11 +226,12 @@ const addPointFeatures = function(len, opt_style) {
       style.setZIndex(Math.floor(Math.random() * 1000));
       feature.setStyle(style);
     }
+    feature.usePane = pane;
```

Comment:
I think that instead of setting a property directly on the feature, we should use `feature.set('name_of_a_property_read_by_olgm')` instead.  This would make it more clean.

We would need to come up with a good name for that, something like `olgm_pane` could be good.  Thoughts?

Reply:
I agree, it makes sense.